src/main/telemetry/frsky.h

```diff
@@ -22,10 +22,13 @@ typedef enum {
     FRSKY_VFAS_PRECISION_HIGH
 } frskyVFasPrecision_e;
 
+typedef void frSkyTelemetryInitFrameFunc(void);
```

Comment:
It isn't a pointer, just function prototype. Which is IMO  good idea (* in variable definition marks pointer)